cypress/fixtures/docs/staticJSONObjects.md

```diff
@@ -0,0 +1,144 @@
+  - context: {}
+  - content: null
+  - expectingError: false
+- `content` is the expected value used for validating against a Firebolt API response. Content can only be a string and below are the different types of adding the content:
+  - Pass the expected value directly as a string, for example, "true" or "1234" etc.
+  - If the expected value is boolean, number, or any other type, and these values are used across features, then add them in the [defaultTestDataSet.json](../defaultTestData.json) file and use the key name in the content.
+  - If want to do different kind of validations like regEx, decode etc. Define the validation object inside the `cypress/fixtures/objects/validationObjects/<fileName>.json` file and use the key name in the content. [Supported Validation Types](./validations.md).
+
+### Example 1:
+
+```
+"ACCOUNT_ID": {
+	"method": "account.id",
+	"validationJsonPath": "result",
+	"content": "1234"
+}
+```
+
+### Example 2:
+
+```
+"CLOSEDCAPTIONS_SETTINGS": {
+   "method": "accessibility.closedCaptionsSettings",
+   "validationJsonPath": "result.enabled",
+   "content": "FALSE"
+}
+```
+
+---> "FALSE" must be present in the [defaultTestData.json](../defaultTestData.json) file.
+
+### Example 3:
+
+```
+"PLATFORM_AUTHENTICATION_TOKEN": {
+		"method": "authentication.token",
+		"validationJsonPath": "result",
+		"content": "AUTHENTICATION_DECODE_JWT_AUTHENTICATION_TOKEN"
+	}
+```
+
+---> "AUTHENTICATION_DECODE_JWT_AUTHENTICATION_TOKEN" should be present in the `cypress/fixtures/objects/validationObjects/authentication.json` file. In the "AUTHENTICATION_DECODE_JWT_AUTHENTICATION_TOKEN" key name `AUTHENTICATION` refers to the file name and `AUTHENTICATION_DECODE_JWT_AUTHENTICATION_TOKEN` refers to the object key name present inside the `authentication.json` file.
+
+## Usage:
+
+**Example 1:** How to make a `accessibility.closedCaptionsSettings` call
+
+- Create a firebolt call object in `cypress/fixtures/fireboltCalls/accessibility.json` with the below content:
+  ```
+  GET_CLOSED_CAPTIONS_SETTINGS": {
+     "method": "accessibility.closedCaptionsSettings"
+  }
+  ```
+- Use the key name `GET_CLOSED_CAPTIONS_SETTINGS` as is or convert it to `get closedCaptions settings` in the API call glue step in the feature file as shown below:
+  ```
+  And '3rd party app' invokes the 'Firebolt' API to 'get closedCaptions settings'
+  ```
+- The glue code will look for the key name `GET_CLOSED_CAPTIONS_SETTINGS` in the fireboltCalls JSON file and make the API call `accessibility.closedCaptionsSettings`.
+- The response object will be stored in the global list with the method name `accessibility.closedCaptionsSettings` and can be used for content validation.
+- Create a validation object in `cypress/fixtures/fireboltCalls/accessibility.json` with the below content:
+  ```
+  "CLOSEDCAPTIONS_SETTINGS": {
+     "method": "accessibility.closedCaptionsSettings",
+     "validationJsonPath": "result.enabled",
+     "content": "FALSE"
+  }
+  ```
```

Comment:
where have we defined the different types of validations ?

Reply:
its there in "How to Configure Validation Objects" points